pennylane/transforms/draw.py

```diff
@@ -152,6 +152,148 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def draw(
+    qnode,
+    wire_order=None,
+    show_all_wires=False,
+    decimals=None,

```

Comment:
Could two be the default here? I had the impression people expect to see the rotation angles, and it helps differentiate the different rotations appearing in the circuit.